aliases/available/docker.aliases.bash

```diff
@@ -0,0 +1,16 @@
+cite 'about-alias'
+about-alias 'docker abbreviations'
+
+alias dklc='docker ps -l'  # List last Docker container
+alias dklcid='docker ps -l -q'  # List last Docker container ID
+alias dklcip="docker inspect `dklcid` | grep IPAddress | cut -d '\"' -f 4"  # Get IP of last Docker container
+alias dkps='docker ps'  # List running Docker containers
+alias dkpsa='docker ps -a'  # List all Docker containers
+alias dki='docker images'  # List Docker images
+alias dkrmac='docker rm $(docker ps -a -q)'  # Delete all Docker containers
+alias dkrmlc='docker-remove-most-recent-container'  # Delete most recent (i.e., last) Docker container
```

Comment:
It's a good question - the current Bash-it code uses a mixture of dash/hyphen and underscores. Deciding to go with one or the other would mean rewriting a lot of it, and potentially breaking it for many people - unless we provide compatibility wrapper.

Having said that, I found a couple of references that recommend against using dash/hyphen as a separator for shell functions:

* http://tldp.org/LDP/abs/html/gotchas.html
* http://unix.stackexchange.com/questions/168221/are-there-problems-with-hyphens-in-functions-aliases-and-executables

Bash seems to support hyphens in function names, so that shouldn't be a problem. How do we decide which character we want to consolidate on? I don't want to start a religious war over this...